api/src/controllers/utils.ts

```diff
@@ -1,12 +1,16 @@
 import argon2 from 'argon2';
 import Busboy from 'busboy';
 import { Router } from 'express';
+import { Worker } from 'node:worker_threads';
+import { file as createTmpFile } from 'tmp-promise';
```

Comment:
Let's implement this ourselves

Reply:
@paescuj 
Both declarations or just the `tmp-promise` part?
I was reusing this since it is already declared on `package.json`, but it looks like it's not being used elsewhere 🤔

Reply:
Ah sorry, meant just the `tmp-promise` package. Hmm odd, I'll check & clean that up 👍 I guess this should be simple enough to implement it ourselves.

Reply:
Thank you for the replacement, @joselcvarela ❤️ Great work, although I was actually thinking more of adding this to the `@directus/utils` package. Not sure if it's worth a separate package 🤔